src/components/networked-hand-tracking.js

```diff
@@ -0,0 +1,34 @@
+﻿/* global AFRAME */
+
+AFRAME.registerComponent('networked-hand-tracking', {
+    dependencies: ['hand-tracking-controls'],
+
+    schema: {
+        options: { default: 'template:#hand-joint-template; attachTemplateToLocal:false' }
```

Comment:
I don't see the hand-joint-template in your PR. What did you use for html template?
Answering this myself looking at your glitch:
```html
<template id="hand-joint-template">
  <a-sphere radius="0.01"></a-sphere>
</template>
```

What about the networked schema? Default networked schema is only syncing position and rotation, did you add scale? I see hand-tracking-controls is setting scale on the joints https://github.com/aframevr/aframe/blob/e0c4ec6d2d07db426d27d76b32ec7aba7d9f5e60/src/components/hand-tracking-controls.js#L250
You probably want to create your own schema that sync position rotation scale with a correct requiresNetworkUpdate function to minimize networked updates, see
https://github.com/networked-aframe/networked-aframe#tracked-controllers-w-synced-gestures

```html
          <a-entity id="left" hand-tracking-controls="hand:left; modelStyle:dots;" networked-hand-tracking="{options: 'template:#hand-joint-template; attachTemplateToLocal:false'}"></a-entity>
```
I didn't know we could use an object like that as attribute value, this is really working?